tests/travis-ci/scripts/run_tests.sh

```diff
@@ -1,66 +1,46 @@
 #!/bin/bash
 
 install_codebase () {
-    mkdir -p ${SITE_DIR}/web/modules
-    cp ${TRAVIS_BUILD_DIR}/tests/travis-ci/fixtures/composer.json.dist ${SITE_DIR}/composer.json
-    cd ${SITE_DIR}
-    perl -i -pe's/\$\{([^}]+)\}/$ENV{$1}/' composer.json
-    COMPOSER_MEMORY_LIMIT=-1 composer install --no-interaction --prefer-dist
+    mkdir -p "${SITE_DIR}/web/modules"
+    cp "${TRAVIS_BUILD_DIR}/tests/travis-ci/fixtures/composer.json.dist" "${SITE_DIR}/composer.json"
+    perl -i -pe's/\$\{([^}]+)\}/$ENV{$1}/' "${SITE_DIR}/composer.json"
+    ln -s "${TRAVIS_BUILD_DIR}" "${SITE_DIR}/web/modules/sparql_entity_storage"
+    cd "${SITE_DIR}" && COMPOSER_MEMORY_LIMIT=-1 composer install --no-interaction --prefer-dist
 }
 
 case "${TEST}" in
     PHPCodeSniffer)
-        cd ${TRAVIS_BUILD_DIR}
+        cd "${TRAVIS_BUILD_DIR}" || exit
```

Comment:
This looks like it will prevent coding standards tests from running when the build folder doesn't exist, but not fail the test. It seems a bit bizarre since the `${TRAVIS_BUILD_DIR}` is expected to always exist, but if this is truly expected to fail, let's exit with an error code like `exit 1`.